bundles/org.openhab.transform.basicprofiles/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="basicprofiles" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>transformation</type>
+	<name>Basic Profiles</name>
+	<description>A set of profiles with basic functionality.</description>
```

Comment:
Can you add connection here also? Unless I missed something, it would be:
```suggestion
	<description>A set of profiles with basic functionality.</description>
	<connection>local</connection>
```